webapp/src/js/services/get-summaries.js

```diff
@@ -88,10 +89,7 @@ angular.module('inboxServices').factory('GetSummaries',
           lineage: getLineage(doc.contact && doc.contact.parent),
           subject: getSubject(doc)
         };
-      } else if (doc.type === 'clinic' ||
-          doc.type === 'district_hospital' ||
-          doc.type === 'health_center' ||
-          doc.type === 'person') { // contact
+      } else if (ContactTypes.includes(doc)) { // contact
```

Comment:
I think this function needs include `doc.contact_type` in it's result, since this will be sent through to LiveList sorting for offline users, with the current algorithm needing access to `contact_type` as well as `type`.

Reply:
Absolutely. Good spotting.